owncloud/owncloud.py

```diff
@@ -804,6 +1022,105 @@ def delete_attribute(self, app, key):
             return True
         raise ResponseError(res)
 
+
+    def get_apps(self):
+    	""" List all enabled apps through the provisioning api.
+	    If this fails, try to run enable_app_ajax('provisioning_api') first.
+	    Returns a dict of apps, with values True/False, representing the enabled state.
+	"""
+	ena_apps = {}
```

Comment:
```
4Spaces def get_…
4Spaces1Tab """ List …
1Tab4Spaces If this …
1Tab4Spaces Returns …
1Tab """
1Tab ena_apps = {}

8Spaces res = …
```

Should this follow PEP8? I pasted this file into PyCharm and it goes wild :frowning: (It's expecting py3 though, since I opened a py3 project)